### PR TITLE
Pyuntl dependency

### LINKED ISSUE #22: pyuntl dependency
Update pyuntl dependency to setup.py after merging the python3 code version into master and get rid of requirements-test.txt

### PR DESCRIPTION
**Pull request by madhuBayy**
@somexpert @ldko This PR fixes #22 . It uses direct dependency url of pyuntl py3 branch.

### CODE REVIEW

**Comment by ldko** [comment]
Should this be changed to be merged into `py3` branch, not master?

**Comment by madhuBayy** [comment]
This should be merged into py3 'cos its using py3 branch of pyuntl for python 3 support.

**Comment by ldko** [comment]
It currently is set to be merged into `master` which is why you can see a bunch of changes on this PR that should not be included, as they are already in `py3` branch.

**Comment by madhuBayy** [comment]
Changed it. Thanks :)

File: README.md [hunk 1]
```diff
@@ -16,16 +16,13 @@ Requirements
 Installation
 ------------
 
-This package is available on the Python Package Index (PyPI), and can be
-installed with pip:
+This application can be installed by following the steps below:
```
Comment (ldko) [inline README.md]: I think we should go back to referring to this as a "package", not an "application".

**Review by ldko — COMMENTED** [review]
This works for me, I have one minor change requested.

File: tox.ini [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
-usedevelop=True
-deps = -r{toxinidir}/requirements/requirements.txt
-       -r{toxinidir}/requirements/requirements-test.txt
+deps = -rrequirements-test.txt
+       pip >= 18.0.1
 commands = pytest
 
 [testenv:py37-flake8]
 deps = flake8
+       pip >= 18.0.1
```
Comment (somexpert) [inline tox.ini]: I don't think we need this here. I believe Python 3.7 comes with a recent-enough version of pip already.

**Review by somexpert — APPROVED** [review]
LGTM. Just have that one comment, but it's not a blocking one.

**Review by ldko — APPROVED** [review]
Looks good to me :+1:

**Pull request merged**